### DISCUSSION TITLE
Reading these analog gas dials

### DISCUSSION THREAD

**Discussion by millskyle** [jomjol/AI-on-the-edge-device#2605] [Q&A]
Anyone have any ideas how to interpret these analog gas dials?  One is labelled .01 cubic metres and the other .05 cubic metres. I have no idea how to read them manually, let alone with this project. 

![Screen Shot 2023-09-13 at 4 15 21 PM](https://github.com/jomjol/AI-on-the-edge-device/assets/7488568/719e3479-261a-4b2d-9868-699a07f04f48)

**Comment by caco3** [comment]
First of all, the project expects that every digit is a factor of 10 to the next smaller/bigger one. So after the last digit (`4`), there usually is a pointer for `0.1` and the next one is for `0.01`.
Example:
![grafik](https://github.com/jomjol/AI-on-the-edge-device/assets/1783586/6d2d6c44-2d37-487f-8608-8ecf8101be01)

Since your meter misses the `0.1` pointer, there is no way you can use the `0.01` or even the `0.05` pointers.
You simply could just use the digits. To my experience, my gas meter uses around `0.5m3` in summer and `2..12m3` in winter per day. So rounding to 1 would give you not a accurate usage per day, but for looking on it per week would be accurate enough.
